### PR TITLE
fix(reply): correct name of reply recipient

### PR DESCRIPTION
**Pull request by Jekrimo**
**What this PR does** 📖
Fix name display in reply bar

**Which issue(s) this PR fixes** 🔨

AP-2154

COMMENTS:
I have no idea why a _snapshots_/getters.test.ts.snap was included. Never touched it, let me know if we need to delete it or anything.

### CODE REVIEW

**Comment by stavares843** [comment]
> I have no idea why a snapshots/getters.test.ts.snap was included. Never touched it, let me know if we need to delete it or anything.


no worries i deleted it 🔨 i also updated the test in order to avoid adding any more snapshots

File: components/views/chat/message/Message.vue [hunk 1]
```diff
@@ -60,6 +60,16 @@ export default Vue.extend({
       return this.$route.params.id
     },
     author(): User {
+      // iridium.users.getUser can not return self, check if it's you first. On reload this would be missing,
```
Comment (josephmcg) [inline components/views/chat/message/Message.vue:63]: the iridium users manager was recently updated to include your own user's details, so this workaround isn't necessary anymore. check `components/views/chat/message/Message.vue` for an example

File: components/views/chat/chatbar/reply/Reply.vue [hunk 2]
```diff
@@ -4,13 +4,21 @@ import Vue from 'vue'
...
   },
   computed: {
     ...mapState(['chat']),
+    replyUserName(): string | undefined {
+      const conversationId = this.$route.params.id
+      if (!conversationId) {
+        return undefined
+      }
+      return iridium.chat.state.conversations[conversationId].name
+    },
```
Comment (josephmcg) [inline components/views/chat/chatbar/reply/Reply.vue:21]: we don't necessary want the conversation name there. We want the name of the message author. you should be able to pass message sender did into user manager.

**Review by josephmcg — CHANGES_REQUESTED** [review]

**Comment by josephmcg** [comment]
fixed here
https://github.com/Satellite-im/Core-PWA/pull/4452

**Pull request closed without merging**